### DISCUSSION TITLE
GeoParquet friendly tileserver

### DISCUSSION THREAD

**Discussion by jwinarske** [maplibre/maplibre-native#2750] [General]
What is the recommendation for tile server with Maplibre and GeoParquet data (Overture)?
Preference is Rust based.  Martin doesn't indicate GeoParquet support.

**Comment by louwers** [comment]
Maybe you can open an issue over at https://github.com/maplibre/martin/issues

@nyurik is probably well aware of the status of GeoParquet in the Rust ecosystem.

**Reply by nyurik** [reply]
GeoParquet is not supported by maplibre either -- but I think there are tools (planetiler?) that can convert geoparquet to pmtiles - and that can be easily served as tiles

**Reply by nyurik** [reply]
@jwinarske per @msbarry, planetiler can do a massive batch conversion of geoparquet to pmtiles, which you can then serve with Martin

**Reply by jwinarske** [reply]
I see.  I need to better understand the gap on maplibre native and GeoParquet

**Reply by louwers** [reply]
@jwinarske This question triggered some discussion on Slack as well.

https://osmus.slack.com/archives/C01G3D28DAB/p1724362213284289

**Comment by jwinarske** [comment]
I need an invite to see that

**Reply by nyurik** [reply]
@jwinarske you don't need an invite - just click https://slack.openstreetmap.us/ - it will auto-send one

**Comment by jwinarske** [comment]
@nyurik thanks

@louwers what channel is your link referencing?

**Reply by louwers** [reply]
![IMG_7687](https://github.com/user-attachments/assets/be0d5e43-d44f-403b-af24-7bf6d766d204)
